docs/docsite/rst/user_guide/windows_winrm.rst

```diff
@@ -566,6 +566,8 @@ encryption uses the more secure TLS protocol instead. If both transport and
 message encryption is required, set ``ansible_winrm_message_encryption=always``
 in the host vars.
 
+Please note message encryption over HTTP is supported on pywinrm 0.3.0 or higher version.
```

Comment:
Instead of putting this inline add another `..Note` further below this alongside the other one saying this.

```
.. Note:: Message encryption over HTTP requires pywinrm>=0.3.0
```

Reply:
@jborean93 Thanks for the review! I modified the commit to mention it as Note